Libraries/LibRequests/Request.cpp

```diff
@@ -37,8 +37,13 @@ void Request::set_request_fd(Badge<Requests::RequestClient>, int fd)
     VERIFY(m_fd == -1);
     m_fd = fd;
 
-    auto notifier = Core::Notifier::construct(fd, Core::Notifier::Type::Read);
+#if defined(AK_OS_WINDOWS)
```

Comment:
answered by the comment on the fixme in the previous commit, but... 

would be nice to hide this ifdef soup in a helper class outside of this method.